### ISSUE TITLE
Dutch version?

### ISSUE THREAD

**Issue by moorlag** [github/training-kit#685]
## Overview
**TL;DR**
Is there interest in a Dutch version of this document?

**Comment by brianamarie** [comment]
Hi @moorlag! It doesn't appear so - would you be interested in creating one?

**Comment by moorlag** [comment]
> Hi @moorlag! It doesn't appear so - would you be interested in creating one?

Yes, think it will help students here to learn the essentials of GitHub more effectively.

**Comment by brianamarie** [comment]
@moorlag That's great! If you're comfortable, please go ahead and create a fork of this repository, add the document and the translation, and open a pull request. Let me know if you'd like help figuring out how to work through any of those steps. 😊

**Comment by moorlag** [comment]
I've started it already. Just need to find someone to proof read it. Any Dutch users you know?

**Comment by brianamarie** [comment]
Yes, in fact! 😄 Once you open up a pull request, I'll `@mention` them there so they can leave a review.

**Comment by chumzee** [comment]
its okay... your get the benefits right 😉😉😉

**Comment by sushmita-m** [comment]
is the dutch version available now?
